### ISSUE TITLE
Support websites with multiple Lightning meta tags

### ISSUE THREAD

**Issue by schneimi** [getAlby/lightning-browser-extension#615]
The Wiki says [`Websites can specify one or more Lightning meta tags.`](https://github.com/getAlby/lightning-browser-extension/wiki/Lightning-Monetization-meta-tag), but this seems to have no effect.

It would be very useful e.g. for a podcast site to be able to make payments to all people involved, maybe even with an option to split the payment.

**Comment by bumi** [comment]
Yes, so far Alby does not fully implement the spec. I would like to first get more feedback on the spec before fully implementing it. 
This is the issue to discuss it: https://github.com/BitcoinAndLightningLayerSpecs/rfc/issues/1 would be great to hear your thoughts there.
